main.pm

```diff
@@ -196,6 +196,11 @@ sub gnomestep_is_applicable() {
     return check_var("DESKTOP", "gnome");
 }
 
+sub snapper_is_applicable() {
+    my $fs = get_var("FILESYSTEM", 'btrfs');
+    return ( $fs eq "btrfs" && get_var("HDDSIZEGB") > 10);
```

Comment:
I'm probably jetlagged, but I don't understand what's going on here , can you explain?